src/pages/Integrations/SplunkSetup/Constants.ts

```diff
@@ -0,0 +1,23 @@
+export const RBAC_GROUPS_API = '/api/rbac/v1/groups/';
+export const RBAC_ROLES_API = '/api/rbac/v1/groups/{}/roles/';
+export const RBAC_ADD_USER_API = '/api/rbac/v1/groups/{}/principals/';
+export const CREATE_INTEGRATION_API = '/api/integrations/v1.0/endpoints';
+export const CREATE_BEHAVIOR_GROUP_API = '/api/notifications/v1.0/notifications/behaviorGroups';
+export const UPDATE_BEHAVIOR_GROUP_API = '/api/notifications/v1.0/notifications/behaviorGroups/{}/actions';
+export const EVENTTYPE_API = '/api/notifications/v1.0/notifications/eventTypes/{}/behaviorGroups';
+
+// If prod use prod UUID, else use stage
+export const NOTIF_ADM_ROLE_UUID = window.insights?.chrome.getEnvironment() === 'prod' ? '' : '15f910ad-150a-46dd-b666-0b21088e9c55';
+export const BEHAVIOR_GROUP_BUNDLE_ID =  window.insights?.chrome.getEnvironment() === 'prod' ? '' : '35fd787b-a345-4fe8-a135-7773de15905e';
+
+export const SPLUNK_GROUP_NAME = 'SPLUNK_INTEGRATION';
+export const SPLUNK_INTEGRATION_NAME = 'SPLUNK_AUTOMATION';
+export const SPLUNK_BEHAVIOR_GROUP_NAME = 'SPLUNK_AUTOMATION_GROUP';
+
+const EVENT_TYPES_PROD = [];
+const EVENT_TYPES_STAGE = [ '6d4d1e03-f1bf-427c-8c62-08a114242b5d', // policy-triggered
+    '95a6d179-5dbd-4de4-a726-5bab2a699912',  // new-recommendation
+    '5df33474-9cfd-40e9-9a86-9f8857147f89', // drift-baseline-detected
+    '75233a10-646c-4e26-a5f1-639a7df8f29f' ]; // resolved-recommendation
```

Comment:
Is it possible to use APIs to grab those ids?

Reply:
technically possible, yes, we provide APIs for that. I'm wondering what's the plan. Are you going to provide a wizard to pick the bundle/events/apps that you want to connect or are always the same?

Reply:
As far I investigated, those UUIDs are the same depending on the environment (stage/prod). The idea was to have those as constants, like this: https://github.com/RedHatInsights/insights-dashboard/blob/master/src/AppConstants.js#L82

Reply:
@jpramos123 I would not count on these to be constant across environments, and I'd definitely avoid hardcoding it. We can hardcode the event types for now.

@josejulio Yes, having an option to select which event type triplets to use is on our future radar. For now (for beta) we would pick some, or maybe leverage wildcards.